crates/spirv-std/src/math_ext.rs

```diff
@@ -82,4 +100,24 @@ impl MathExt for f32 {
     fn copysign(self, sign: f32) -> f32 {
         unsafe { core::intrinsics::copysignf32(self, sign) }
     }
+
+    fn step(self, edge: f32) -> f32 {
```

Comment:
this should probably be an intrinsic that compiles to the spir-v step instruction, but whatever, who cares right now. I agree with the `x.step(edge)` order (and Freya does too ✨).

Reply:
afaik `step` is almost certainly compiled down to an `if` like this anyway so ✨